<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ExchangeWidget</class>
    <widget class="QWidget" name="ExchangeWidget">
        <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
                <widget class="QLabel" name="label">
                    <property name="text">
                        <string>Amount:</string>
                    </property>
                </widget>
            </item>
            <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="amount">
                    <property name="minimum">
                        <double>0.010000000000000</double>
                    </property>
                    <property name="maximum">
                        <double>1000000.000000000000000</double>
                    </property>
                    <property name="value">
                        <double>100.000000000000000</double>
                    </property>
                </widget>
            </item>
            <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                    <property name="text">
                        <string>From:</string>
                    </property>
                </widget>
            </item>
            <item row="1" column="1">
                <widget class="QComboBox" name="fromCurrency"/>
            </item>
            <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                    <property name="text">
                        <string>To:</string>
                    </property>
                </widget>
            </item>
            <item row="2" column="1">
                <widget class="QComboBox" name="toCurrency"/>
            </item>
            <item row="3" column="0" colspan="2">
                <widget class="QPushButton" name="convertButton">
                    <property name="text">
                        <string>Convert</string>
                    </property>
                </widget>
            </item>
            <item row="4" column="0" colspan="2">
                <widget class="QLabel" name="resultLabel">
                    <property name="text">
                        <string>Result will be shown here</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">font-size: 16pt; font-weight: bold;</string>
                    </property>
                </widget>
            </item>
            <item row="5" column="0" colspan="2">
                <widget class="QPushButton" name="requestButton">
                    <property name="text">
                        <string>Create Exchange Request</string>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
